src/zenml/integrations/skypilot/orchestrators/skypilot_base_vm_orchestrator.py

```diff
@@ -271,29 +272,36 @@ def prepare_or_run_pipeline(
                 f"sudo docker login --username $DOCKER_USERNAME --password "
                 f"$DOCKER_PASSWORD {stack.container_registry.config.uri}"
             )
-            task_envs = {
-                "DOCKER_USERNAME": docker_username,
-                "DOCKER_PASSWORD": docker_password,
-            }
+            task_envs["DOCKER_USERNAME"] = (docker_username,)
+            task_envs["DOCKER_PASSWORD"] = (docker_password,)
         else:
             setup = None
-            task_envs = None
 
         # Run the entire pipeline
 
         # Set the service connector AWS profile ENV variable
         self.prepare_environment_variable(set=True)
 
         try:
+            if isinstance(self.cloud, sky.clouds.Kubernetes):
+                image = image
+                run_command = f"/opt/venv/bin/{entrypoint_str} {arguments_str}"
+                setup = None
+                down = False
+                idle_minutes_to_autostop = None
```

Comment:
These options dont work anymore then in this mode? We should add to the docs

Reply:
Yes will do add them to sky/k8s doc